src/main/webapp/WEB-INF/jsp/sign-on-web3.jsp

```diff
@@ -140,7 +126,7 @@
             with, because you can use your Web3 account instead.
         </p>
 
-        <iframe style="border-radius: 8px;" width="100%" height="160" src="https://www.youtube.com/embed/YVgfHZMFFFQ" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+        <!-- <iframe style="border-radius: 8px;" width="100%" height="160" src="https://www.youtube.com/embed/YVgfHZMFFFQ" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe> -->
```

Comment:
Do we need to hide the YouTube video? :-)

Reply:
Oh, my bad :D not actually!

Reply:
Ok, yeah, I think it shouldn't cause any problems because it's loaded over `https`. Can we restore it?